### DISCUSSION TITLE
关于停止解盘的公告

### DISCUSSION THREAD

**Discussion by SylarLong** [SylarLong/iztro#151] [Announcements]
经过思考决定停止申请解盘的活动，诚如 [这位老师](https://github.com/SylarLong/iztro/discussions/124#discussioncomment-7881310) 在帖子里说的情况，未经过定盘的解盘是毫无意义的。再加上自己一些时间上的关系，所以做出了这个决定。还没有解盘的各位如果仍然有解盘需求，请在profile页面添加我的微信。我会在空余时间帮大家定盘以后在进行解盘。

所有未解盘的issue我将会关闭，实在抱歉，感谢大家对iztro的支持。

如果有需要删除issue的朋友请在下面评论，我会帮大家删除。

**Comment by Capchen** [comment]
希望补充下「紫微斗数四化」的文档

**Reply by SylarLong** [reply]
后面的文档会陆续更新的，感谢关注！
